src/js/components/devices/authorized-devices.js

```diff
@@ -265,7 +265,6 @@ export const Authorized = ({
     if (!acceptedCount) {
       return;
     }
-    dispatch(advanceOnboarding(onboardingSteps.DEVICES_ACCEPTED_ONBOARDING));
```

Comment:
did you check if there is another place doing dispatch(advanceOnboarding(onboardingSteps.DEVICES_ACCEPTED_ONBOARDING));?
otherwise step might not be changed

Reply:
Yes, in order to proceed, the user should click a button on the tooltip with a hint, that says "Yes, let's deploy!", or, alternatively click on "Dismiss the  tutorial".